### ISSUE TITLE
Documentation: Supporting new devices.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [olikraus/ucglib#43]
```
Reference to a callback function (ucg_com_my_own_hal), independent of *_ilsoft 
function (ucg_dev_ssd1351_18x128x128_ilsoft), is made at 
http://code.google.com/p/ucglib/wiki/hal . However, no corresponding function 
is obvious in the codebase at present. Rather, the documented functionality 
seems implemented by the *_ilsoft function.

A related function, ucg_ext_ssd1351_18(), has been discovered to address only 
msg == UCG_MSG_DRAW_L90SE . This functionality does not seem critical, and may 
have previously been handled by ucg_dev_ssd1351_18x128x128_ilsoft() through 
ucg_dev_ic_ssd1351_18() .

Ultimately, the important function/procedure calls seem to take place in the 
following order, by example:
Ucglib4WireHWSPI --> Ucglib_SSD1351_18x128x128_HWSPI (Ucglib.h declaration) --> 
ucg_dev_ssd1351_18x128x128_ilsoft (ucg_dev_oled_128x128_ilsoft.c definition)

In other words, for device SSD1351, all essential hardware-abstraction-layer 
callback functionality seems handled by ucg_dev_ssd1351_18x128x128_ilsoft.
```

Original issue reported on code.google.com by `spamfree...@gmail.com` on 20 Jan 2015 at 11:13

**Comment by olikraus** [comment]
Not an issue, closing
